jsonschema-validation.xml

```diff
@@ -1028,6 +1028,27 @@
                 </t>
             </section>
 
+            <section title='"deprecated"'>
+                <t>
+                    The value of this keyword MUST be a boolean.  When multiple occurrences
+                    of this keyword are applicable to a single sub-instance, the resulting
+                    value MUST be true if any occurrence specifies a true value, and MUST
+                    be false otherwise.
+                </t>
+                <t>
+                    If "deprecated" has a value of boolean true, it indicates that applications
+                    SHOULD refrain from usage of the declared operation. It MAY mean the property
+                    is going to be removed in the futuree.
+                </t>
+                <t>
+                    An instance document that is marked as "deprecated" for the entire document
```

Comment:
I would add something about how this is done by setting `"deprecated": true` in the root schema.  Just reading through this without thinking of the history, it sounds a bit like marking an instance document as deprecated is a rather separate thing.